src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected int taskID;
+    protected String description;
+    protected boolean isDone;
```

Comment:
Consider writing getter methods instead of directly referencing these attributes in child classes.